src/components/Blocking/ReportUser.js

```diff
@@ -105,7 +105,7 @@ const styles = StyleSheet.create({
     marginLeft: 21
   },
   desc: {
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color change from `COLORS.blackgrey` to `COLORS.gray400` seems fine if it aligns with the new design requirements. However, please ensure that this color change doesn't affect readability and accessibility standards (WCAG 2.0 level AA requires a contrast ratio of at least 4.5:1 for normal text).
```diff
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->